iothub/service/tests/DeserializationTests.cs

```diff
@@ -0,0 +1,303 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Tests
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class DeserializationTests
```

Comment:
For serialization tests, I'd recommend creating an object, serializing it, then deserializing it. You can then compare the before and after. It validates both the serialization and deserialization steps.